webpack/redux/reducers/RedHatRepositories/enabled.js

```diff
@@ -31,6 +31,7 @@ const mapRepo = (repo) => {
     orphaned: repo.product.orphaned,
     contentId: parseInt(repo.content_id, 10),
     productId: parseInt(repo.product.id, 10),
+    canDisable: repo.content_view_versions.length === 0,
```

Comment:
Actually one final comment.. May be better to use `size` or `count` here, in case there are a lot of content view versions..

Reply:
Talked about this offline and resolving comment based on your blessings! 👼🏼